.idea/modules.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/CBOnlineApp.iml" filepath="$PROJECT_DIR$/.idea/CBOnlineApp.iml" />
-      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
-      <module fileurl="file://$PROJECT_DIR$/onlineapi/onlineapi.iml" filepath="$PROJECT_DIR$/onlineapi/onlineapi.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
+    <component name="ProjectModuleManager">
```

Comment:
do not commit this file

Reply:
won't this be added in gitignore?

Reply:
I've updated gitignore you must rebase first

Reply:
Please guide me for the same

Reply:
https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository